packages/UsageAnalysis/files/Test Track/Viewers/Trellis plot.md

```diff
@@ -1,3 +1,51 @@
+### Trellis plot
+
+1. Open the **demog** dataset.
```

Comment:
Link dataset to the test case

Reply:
added     "datasets": ["System:DemoFiles/demog.csv","System:DemoFiles/SPGI.csv"]